platform/o.n.swing.laf.flatlaf/src/org/netbeans/swing/laf/flatlaf/FlatLFCustoms.java

```diff
@@ -63,6 +67,12 @@
             "nb.multitabs.button.left.icon", FlatTabControlIcon.get(TabControlButton.ID_SCROLL_LEFT_BUTTON), // NOI18N
             "nb.multitabs.button.right.icon", FlatTabControlIcon.get(TabControlButton.ID_SCROLL_RIGHT_BUTTON), // NOI18N
             "nb.multitabs.button.rollover", true, // NOI18N
+
+            // Change some colors from ColorUIResource to Color because they are used as
+            // background colors for checkboxes (e.g. in org.netbeans.modules.palette.ui.CategoryButton),
+            // which in FlatLaf paint background only if background color is not a UIResource.
+            "PropSheet.setBackground", new Color(UIManager.getColor("PropSheet.setBackground").getRGB()),
+            "PropSheet.selectedSetBackground", new Color(UIManager.getColor("PropSheet.selectedSetBackground").getRGB()),
```

Comment:
Am I right that these changes (in FlatLFCustoms) are related to the LAF's appearance as a whole, i.e. not specifically related to the form designer?

Reply:
Yes.

The controlFont is also used in other places. E.g. as fallback font in `HtmlRenderer`. The "Terminal" uses the size of controlFont to create its monospaced font for "Terminal" view and "Options > Misc. > Terminal > Preview".

The `PropertySheet` component is also used outside of the form designer.

Reply:
I don't see a change in the property sheet checkbox rendering in the before/after screenshots. Am I missing something?

Reply:
The checkbox is in the designer palette, but it does not look like a checkbox because the check-icons are changed:

![image](https://user-images.githubusercontent.com/5604048/71833650-ac838f80-30ad-11ea-928b-0e8d02c5b728.png)

Reply:
Ah! Tricky use of JCheckBox there...